### ISSUE TITLE
[Bug]: Carousel indicator is missing aria-label even when set through indicatorsLabelFormat property

### ISSUE THREAD

**Issue by mddragnev** [IgniteUI/igniteui-webcomponents#1942]
### Which component(s) are affected?

Carousel

### Description

Carousel indicator should have aria-label as per `indicatorsLabelFormat` property docs. It does not have even if the input is set to another value

### Reproduction

Go to the storybook and on the carousel set some string to the `indicatorsLabelFormat` property. See that no aria-label is set to the indicators.

### Workaround

No workaround

### Is this a regression?

No or unsure. This never worker, or I haven't tried before.

### Affected versions

master

### Browser/OS/Node environment

n/a
